sematic/ui/src/pipelines/RunDetailsPanel.tsx

```diff
@@ -74,14 +76,10 @@ export function RunDetailsPanel() {
                 <Typography variant="h6">{selectedRun.name}</Typography>
                 <Typography fontSize="small" color="GrayText" component="span">
                   <code style={{ fontSize: 12 }}>
-                  ID: {selectedRun.id}
+                  ID: {selectedRun.id} <CopyButton text={selectedRun.id} message="Copied run ID" />
                   {
                     selectedRun.original_run_id &&
-                    /*
-                      TODO #278: replace full id with a 6-character trimmed
-                      link using "./Notes/RunId"
-                    */
-                    <> (cloned from {selectedRun.original_run_id})</>
+                    <> (cloned from <RunId runId={selectedRun.original_run_id} copy={false} />)</>
```

Comment:
Please write "cloned from" in non-formatted font, and maybe drop the parentheses.

Reply:
Sure, thanks for the suggestion. How does this look?

<img width="803" alt="image" src="https://user-images.githubusercontent.com/1046489/213329867-1fbb0e4c-3e67-428f-900f-e79d3a04f22f.png">